skietbaan_fe/src/components/NavbarMenu.scss

```diff
@@ -0,0 +1,89 @@
+
+/* .fixed-top{
+    position:relative ;
+  }
+
+  .navbar .navbar-custom{
+    background-color: #f0f0f0;
+    color: #f0f0f0;
+  }
+
+  .App div .navbar {
+    margin: 0%;
+  }
+
+  .App div .navbar a{
+    color: #f0f0f0;
+  }
+
+  .App div .navbar .fa{
+    display: grid;
+    text-align: center;
+   }
+
+  .App div .navbar .fa:hover{
+    align-content: center
+   }
+
+  .App div .navbar button{
+    padding: 0%;
+    width: fit-content;
+    color: #f0f0f0;
+    background-color: #f0f0f0;
+  }
+*/
+  .dropdown-menu.show{
+    background-color: #f0f0f0;
+    border: none;
+    border: 1px solid  #f0f0f0;
+  }
+
+  .leaderboard-icon-grey{
+    height: 40px;
+    width: 59px;
+  }
+
+  .add-score-icon-grey{
+    height: 40px;
+    width: 49px;
+  }
+  
+  .create-icon-grey{
+    height: 40px;
+    width: 31px;
+  }
+  
+  .docs-icon-grey{
+    height: 40px;
+    width: 54px;
+  }
+
+  .notifications-icon-grey{
+    height: 40px;
+    width: 57px;
+  }
+
+  .profile-icon-grey{
+    height: 40px;
+    width: 30px;
+  }
+
+  .more-icon-grey{
+    height: 38px;
+    width: 28px;
+  }
+  
+  .second-row-navbar{
+    display: none;
+
```

Comment:
cleanup empty lines 78 and 80 also empty line EOF

Reply:
did that